src/main/java/net/vulkanmod/mixin/window/WindowMixin.java

```diff
@@ -70,6 +69,8 @@ private GLCapabilities redirect2() {
     @Inject(method = "<init>", at = @At(value = "INVOKE", target = "Lorg/lwjgl/glfw/GLFW;glfwCreateWindow(IILjava/lang/CharSequence;JJ)J"))
     private void vulkanHint(WindowEventHandler windowEventHandler, ScreenManager screenManager, DisplayData displayData, String string, String string2, CallbackInfo ci) {
         GLFW.glfwWindowHint(GLFW_CLIENT_API, GLFW_NO_API);
+        //Fix Gnome Client-Side Decorators
+        GLFW.glfwWindowHint(GLFW_DECORATED, VideoResolution.isGNOME() && VideoResolution.isWayLand() ? GLFW_FALSE : GLFW_TRUE);
```

Comment:
```suggestion
        GLFW.glfwWindowHint(GLFW_DECORATED, VideoResolution.isWayLand() ? GLFW_FALSE : GLFW_TRUE);
```

Reply:
Although I'm not too sure if this error exists on X11 as I only use Wayland. I'll see to check it out later today / tomorrow.